service/controller/v11/resource/deployment/template/public_load_balancer_setup.json

```diff
@@ -30,6 +30,12 @@
     },
     "ports":{
       "type":"array"
+    },
+    "UDPPorts":{
+      "type":"array",
+      "defaultValue": [
+        { "frontend": "1234", "backend": "1234" }
```

Comment:
May be a dumb question but why it's 1234?

Reply:
It's just a random port.